Dockerfiles/agent/entrypoint/01-check-apikey.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Don't allow starting without an apikey set
-if [[ -z "${DD_API_KEY}" ]]; then
+if [[ -z "${DD_API_KEY}" && ! -r "${DD_API_KEY_FILE}" ]]; then
     echo ""
     echo "=================================================================================="
     echo "You must set an DD_API_KEY environment variable to run the Datadog Agent container"
```

Comment:
We should update the comment to reflect the ability to use the new env var to read the key from a file.

Reply:
Comment updated.